app/controllers/settings/exports/blocked_domains_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 module Settings
   module Exports
-    class BlockedDomainsController < ApplicationController
+    class BlockedDomainsController < BaseController
```

Comment:
ok, if this inherits from Settings::BaseController, doesn't that still require_functional!?